test/tests/functional/pbs_node_rampdown.py

```diff
@@ -6420,7 +6420,7 @@ def test_release_job_array(self):
         # Below variable is being used for the accounting log match
         # which is currently blocked on PTL bug PP-596.
         # new_exec_host_esc = self.job1_exec_host.replace(
-        # "*", "\*").replace("[", "\[").replace("]", "\]").replace("+", "\+")
+        # "*", r"\*").replace("[", r"\[").replace("]", r"\]").replace("+", r"\+")
```

Comment:
pep8 check is failing on this:
```
E501 line too long (81 > 79 characters)
        # "*", r"\*").replace("[", r"\[").replace("]", r"\]").replace("+", r"\+")
                                                                             ^
```
you can see the failure for yourself on azure pipelines. please fix the CI before we proceed to merge this.

Reply:
There are a bunch of too-long (E501) lines, and it is rather a work of formatter, yapf or black.

Reply:
Can you push the changes once it is done?

Reply:
Done.